packages/flutter_tools/lib/src/commands/test.dart

```diff
@@ -330,6 +330,18 @@ class TestCommand extends FlutterCommand with DeviceBasedDevelopmentArtifacts {
     return super.verifyThenRunCommand(commandPath);
   }
 
+  WebRendererMode get webRenderer {
+    final String? webRendererString = stringArg('web-renderer');
+    final WebRendererMode mode = webRendererString != null
+      ? WebRendererMode.values.byName(webRendererString)
+      : WebRendererMode.auto;
+    if (mode == WebRendererMode.auto && useWasm) {
+      // Wasm defaults to skwasm
+      return WebRendererMode.skwasm;
+    }
+    return mode;
+  }
```

Comment:
This method is duplicated between here and `commands/run.dart`, maybe the implementation could be shared to a `WebRendererMode getRendererMode(String? userString, bool useWasm = false);`? It could even be a `static` in the `WebRendererMode enum` itself?

Reply:
Yeah that's a good idea, I'll do that.